### DISCUSSION TITLE
가로모드로 만든 맵에서, 세로모드 사진을 찍고싶습니다. 방법이 없을까요?

### DISCUSSION THREAD

**Discussion by ted002** [naverz/zepeto-studio-kor#530] [5. Q&A]
현재 피드는 세로모드 최적화로 만들어져 있습니다. 그래서 제페토 자체의 셀카모드로도 세로모드가 따로 있죠.
하지만 현재 만들다보니 제페토에서 제공한 방법으로는 세로모드를 만드는게 참 쉽지 않더군요
혹시 가로모드에서도 세로모드로 사진을 찍을 수 있는 방법이 없을까요?

**Comment by Zepeto-TechEvangelist** [comment]
스크린삿을 세로로 촬영하기 위해서는 아래와 같이 렌더텍스처의 사이즈와 RawImage를 수정하시면 세로로 촬영할 수 있습니다.

1. 렌더텍스처의 사이즈를 세로 비율로 변경합니다.

![image (2)](https://user-images.githubusercontent.com/100118650/160081439-f79b164c-b150-485e-a780-943b38a37f33.png)

2. 출력되는 rawimage의 rect transform을 세로 비율로 변경합니다.
3. 
![image (3)](https://user-images.githubusercontent.com/100118650/160081493-23eb8b98-bca4-4223-a6a5-dcd767ce6bb7.png)


아래는 스크린샷 출력 결과입니다. 저장/공유/피드 생성 시 지정한 렌더텍스처 사이즈로 저장 및 전송됩니다.

![image (4)](https://user-images.githubusercontent.com/100118650/160081553-d4a1c693-ee8d-4aaf-a071-2898a520366c.png)


추가적으로 셀카모드 예제의 경우 이후 업데이트를 통해 방향 전환이 제공될 예정입니다. 😄
